src/components/DetailItems/NameDetailItem.tsx

```diff
@@ -0,0 +1,29 @@
+import type { FC } from 'react';
+import { DetailsItem } from 'src/modules/Providers/utils/components/DetailsPage/DetailItem';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { getName } from '@utils/crds/common/selectors';
+
+import type { ResourceDetailsItemProps } from './utils/types';
+
+const NameDetailsItem: FC<ResourceDetailsItemProps> = ({ helpContent, moreInfoLink, resource }) => {
+  const { t } = useForkliftTranslation();
+
+  const defaultMoreInfoLink =
```

Comment:
Please create, if it does not exist, a central place for all links and move it there